packages/worker/src/sdk/users/users.ts

```diff
@@ -14,11 +18,13 @@ export async function invite(
   const matchedEmails = await usersCore.searchExistingEmails(
     users.map(u => u.email)
   )
-  const newUsers = []
+  const newUsers: InviteUserRequest[] = []
 
   // separate duplicates from new users
   for (let user of users) {
     if (matchedEmails.includes(user.email)) {
+      // This "Unavailable" is load bearing. The tests and frontend both check for it
```

Comment:
This is sadly true in a few places - there are a few tests that utilise events to check whether certain outcomes have occurred, rather than using API responses - but luckily there isn't too many cases like this.